src/Insight.ts

```diff
@@ -93,6 +93,21 @@ export class Insight {
 
     return Math.ceil(feeRate / 1024)
   }
+
+  public async getTransactionInfo(id: string): Promise<Insight.ITransactionInfo> {
+    const res = await this.axios.get(`/tx/${id}`)
+    const rawInfo = res.data as Insight.IRawTransactionInfo
+    const { txid, confirmations, time, vin: [{ vout, addr }, ...rest], vout: [{ value }, ...other] } = rawInfo
+
+    return {
+      txid,
+      confirmations,
+      time: new Date(time * 1000),
+      type: vout === 0 ? "in" : "out",
```

Comment:
it's better to use "vin" and "vout"

Reply:
also, maybe it's better to not change the API output.

let view handle the conversion between number and `Date`

Reply:
just return the API output as is